src/graphorm/index.ts

```diff
@@ -98,7 +98,9 @@ const createSmartTitleField = ({ field }: { field: string }): GraphORMField => {
         userId: ctx.userId!,
       });
 
-      const i18nValue = typedParent.i18nTitle?.[ctx.contentLanguage];
+      const i18nValue =
+        ctx.contentLanguage && typedParent.i18nTitle?.[ctx.contentLanguage];
+
```

Comment:
```suggestion
      const i18nValue =
        ctx.contentLanguage ? typedParent.i18nTitle?.[ctx.contentLanguage] : undefined

```

just to make it more specific